src/User/Form/LoginForm.php

```diff
@@ -117,6 +117,7 @@ public function login()
     {
         if ($this->validate()) {
             $duration = $this->rememberMe ? $this->module->rememberLoginLifespan : 0;
+            $this->user->updateAttributes(['last_login_at' => time()]);
```

Comment:
I think this is not the place to do this, as the login may failed on `return Yii::$app->getUser()->login($this->user, $duration);`. The proper place to do this is on the `SecurityController` after the login has been successful and prior to raise the `EVENT_AFTER_LOGIN`: 

```php 
/**
     * Controller action responsible for handling login page and actions.
     *
     * @return array|string|Response
     */
    public function actionLogin()
    {
        if (!Yii::$app->user->getIsGuest()) {
            return $this->goHome();
        }

        /** @var LoginForm $form */
        $form = $this->make(LoginForm::class);
        /** @var FormEvent $event */
        $event = $this->make(FormEvent::class, [$form]);

        if (Yii::$app->request->isAjax && $form->load(Yii::$app->request->post())) {
            Yii::$app->response->format = Response::FORMAT_JSON;

            return ActiveForm::validate($form);
        }

        if ($form->load(Yii::$app->request->post())) {
            $this->trigger(FormEvent::EVENT_BEFORE_LOGIN, $event);
            if ($form->login()) {

                Yii::$app->getUser()->identity->updateAttributes(['last_login_at' => time()]);

                $this->trigger(FormEvent::EVENT_AFTER_LOGIN, $event);

                return $this->goBack();
            }
        }

        return $this->render(
            'login',
            [
                'model' => $form,
                'module' => $this->module,
            ]
        );
    }
```